buildandtest.yml

```diff
@@ -44,13 +41,11 @@ steps:
     projects: |      
      $(Build.SourcesDirectory)\sln\OData.Pipeline.sln

```

Comment:
Are the e2e tests included back to OData.Pipeline.sln?

Reply:
Yes